contents/docs/_snippets/identify-frontend-code.mdx

```diff
@@ -8,10 +8,13 @@ posthog.identify(
 ```
 
 ```Android
-PostHog.with(this)
-     .identify(distinctID, new Properties() // Replace 'distinctID' with your user's unique identifier
-        .putValue("name", "Max Hedgehog") // optional: set additional user properties
-        .putValue("email", "max@hedgehogmail.com));
+PostHog.identify(
```

Comment:
Since the language here is `Android`, is that gonna be rendered as `java`? we need to make it `kotlin`.

Reply:
Kotlin can be used on Java BE and Android FE, but we have one SDK for each, so it wont work.
Also, this is the platform tab, and not the language used.

Reply:
Thanks